core/src/main/scala/io/udash/properties/single/DirectProperty.scala

```diff
@@ -18,4 +18,6 @@ private[properties] class DirectPropertyImpl[A](val parent: ReadableProperty[_])
 
   override def touch(): Unit =
     valueChanged()
+
+  override def toString: String = s"DirectProperty($value)"
```

Comment:
It was, I just wanted to make the test failures (and other similar use cases) more sensible to read without a debugger. It's not so straightforward with the other ones to avoid a side-effect.